tests/phpunit/tests/multisite/cleanDirsizeCache.php

```diff
@@ -234,14 +239,32 @@ function _filter_pre_recurse_dirsize() {
 
 		function _get_mock_dirsize_cache_for_site( $site_id ) {
 			return array(
-				"wp-content/uploads/sites/$site_id/2/2" => 22,
-				"wp-content/uploads/sites/$site_id/2/1" => 21,
-				"wp-content/uploads/sites/$site_id/2"   => 2,
-				"wp-content/uploads/sites/$site_id/1/3" => 13,
-				"wp-content/uploads/sites/$site_id/1/2" => 12,
-				"wp-content/uploads/sites/$site_id/1/1" => 11,
-				"wp-content/uploads/sites/$site_id/1"   => 1,
-				"wp-content/uploads/sites/$site_id/custom_directory" => 42,
+				"/uploads/sites/$site_id/2/2" => 22,
+				"/uploads/sites/$site_id/2/1" => 21,
+				"/uploads/sites/$site_id/2"   => 2,
+				"/uploads/sites/$site_id/1/3" => 13,
+				"/uploads/sites/$site_id/1/2" => 12,
+				"/uploads/sites/$site_id/1/1" => 11,
+				"/uploads/sites/$site_id/1"   => 1,
+				"/uploads/sites/$site_id/custom_directory" => 42,
+			);
+		}
+
+		/*
+		 * todo add desc, covers, etc
+		 *
+		 *
+		 * @ticket 51913
+		 */
+		function test_5_5_transient_structure_compat() {
```

Comment:
I can push some tests up for this so you might need to pull the branch in the morning.

Reply:
👍🏻  thanks!